examples/nitclk/synchronize-tclk.py

```diff
@@ -0,0 +1,167 @@
+# Demonstrates how to configure multiple NI Arbitrary Waveform Generators to generate synchronized waveforms with TClk
+# 
+# The gRPC API is built from the C API. NI-FGEN documentation is installed with the driver at:
+# C:\Program Files (x86)\IVI Foundation\IVI\Drivers\niFgen\documentation\English\SigGenHelp.chm
+#
+# The gRPC API is built from the C API. NI-TClk documentation is installed with the driver at:
+# C:\Program Files (x86)\IVI Foundation\IVI\Drivers\niTClk\documentation\English\nitclk.chm
+#
+# Getting Started:
+#
+# To run this example, install "NI-FGEN Driver" on the server machine.
+# Link : https://www.ni.com/en-us/support/downloads/drivers/download.ni-fgen.html
+#
+# For instructions on how to use protoc to generate gRPC client interfaces, see our "Creating a gRPC Client" wiki page.
+# Link: https://github.com/ni/grpc-device/wiki/Creating-a-gRPC-Client
+#
+# Running from command line:
+#
+# Server machine's IP address and port number can be passed as separate command line arguments.
+#   > python synchronize-tclk.py <server_address> <port_number>
+# If they are not passed in as command line arguments, then by default the server address will be "localhost:31763"
+
+import grpc
+import sys
+import nifgen_pb2 as nifgen_types
+import nifgen_pb2_grpc as grpc_nifgen
+import nitclk_pb2 as nitclk_types
+import nitclk_pb2_grpc as grpc_nitclk
+import session_pb2 as session_types
```

Comment:
I don't see either of session imports being used anywhere. Do you need these?

Reply:
removed